### DISCUSSION TITLE
How to I change the default image?

### DISCUSSION THREAD

**Discussion by ashishb** [nektos/act#2407] [General]
I mistakenly set it to `full` how do I change this to medium?
I don't have `~/.config/act/actrc` on Mac OS.
I don't even have `act` directory in my `~/.config`

**Comment by ashishb** [comment]
Found it in `~/Library/Application Support/act/actrc`

**Comment by celarye** [comment]
It is quite annoying to figure this out, the documentation leaves much to be desired. Thank you for coming back and answering yourself.
